lib/foreman_tasks/dynflow.rb

```diff
@@ -117,5 +121,15 @@ def eager_load_actions!
     def loaded_paths
       @loaded_paths ||= Set.new
     end
+
+    def after_initialize(&block)
```

Comment:
@ShimShtein why is that an issue? I expect we want to apply that only for processes that acually run the executor, so the on_init is actually a better fit